docs/hugo/content/design/ADR-2022-09-Multiple-Credential-Operator.md

```diff
@@ -0,0 +1,153 @@
+---
+title: '2022-09: Support For Multiple Credentials Under Global Operator'
+---
+
+## Context
+
+The operator needs to support creating Azure resources using multiple credentials, managed by different groups, within one cluster by a single global instance of operator. 
```

Comment:
Suggest inverting the structure of the current intro to start with the problem being solved, rather than the change we're making to ASO.

Maybe 

> Some users of Azure Service Operator want to have a single instance that manages Azure resources across multiple subscriptions. This is useful, for example, if the organisation has a large cluster used by multiple teams, each team having their own Azure subscription. 
> To support this, ASO will introduce support for multiple credentials, each associated with a specific scope:
> * *Namespaced Secrets* will be used for all Azure Resources within a specific cluster namespace
> * *ResourceGroup Secrets* will be used for all Azure Resources within a specific Resource Group
> * *Resource Secrets* will be used for specific resources

Reply:
Using @theunrepentantgeek's suggestion here. FYI @matthchr

Reply:
Going to resolve this, as it looks like you've taken this feedback.